app/services/proofing/mock/state_id_mock_client.rb

```diff
@@ -12,8 +12,8 @@ class StateIdMockClient < Proofing::Base
       stage :state_id
 
       SUPPORTED_STATES = %w[
-        AR AZ CO CT DC DE FL GA IA ID IL IN KY MA MD ME MI MO MS MT ND NE NJ NM PA
-        RI SC SD TX VA VT WA WI WY
+        AR AZ CO CT DC DE FL GA IA ID IL IN KS KY MA MD ME MI MO MS MT NC ND NE
+        NJ NM OH OR PA RI SC SD TN TX VA VT WA WI WY
       ].to_set.freeze
```

Comment:
Now that the mock proofer and real proofer are in the same codebase, what if we just referenced that constant instead of duplicating here?

```
Steps::VerifyBaseStep::AAMVA_SUPPORTED_JURISDICTIONS
```

Reply:
good call, willfix